hapic/ext/pyramid/context.py

```diff
@@ -114,13 +114,24 @@ def get_file_response(self, file_response: HapicFile, http_code: int):
             response.content_type = file_response.mimetype
             response.app_iter = FileIter(file_response.file_object)
 
+        response.conditional_response = file_response.use_conditional_response
```

Comment:
Shouldn't the `file_response` parameter be `hapic_file/file` instead ? The "real" `FileResponse` object is named `response` which is confusing. (I know this is not coming from your PR so feel free to ignore this comment).

Reply:
That's a good suggestion, but i clearly prefer to to change such things in this PR.